doc/man/8/rados.rst

```diff
@@ -97,6 +97,8 @@ Pool specific commands
 
 :command:`put` *name* *infile* [--offset offset]
   Write object name with start offset (default:0) to the cluster with contents from infile.
+  
+  **Warning:** Please be aware that this directly creates a single object at the RADOS layer directly. This can interfere with data distribution across the OSDs and even overload the cluster when trying to replicate it. Consider using the frontend protocols like S3, RBD, CephFS instead, which all stripe the data properly.
```

Comment:
if you create an erasure pool and store an object using "rados put", is it not striping the data? or is it doing it in a not proper way? I think the last part of the line could be a little confusing